src/redux/reducers/profileReducer.js

```diff
@@ -0,0 +1,27 @@
+import * as actionTypes from "../action_types";
+import initialState from "./initialState";
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case actionTypes.PROFILE_LOADING:
+      return {
+        ...state,
+        loading: true
+      };
+    case actionTypes.GET_PROFILE:
+      return {
+        ...state,
+        profile: action.payload,
+        loading: false
+      };
+    case actionTypes.CLEAR_CURRENT_PROFILE:
```

Comment:
Same case in this file. Import the specific action-types you need. Instead of having to type out `actionTypes.GET_PROFILE` everywhere.

Reply:
@kimotho-njoki `profiles` is redundant. I will remove it.

Reply:
@kimotho-njoki Noted.